Makefile

```diff
@@ -53,21 +54,21 @@ check-licenses: fetch
 		"$(BUILDER_IMAGE)" \
 		bash -c "cargo deny --all-features check --disable-fetch licenses bans sources"
 
-# Builds, Lints and Tests the Rust workspace
-build: fetch check-licenses
+# Builds, Lints, and Tests the Rust workspace locally
+build: check-licenses
 	cargo fmt -- --check
-	cargo build --locked
 	cargo test --locked
+	cargo build --locked
 
-brupop-image: fetch check-licenses
+# Builds only the brupop image. Useful target for CI/CD, releasing, etc.
+brupop-image:
 	docker build $(DOCKER_BUILD_FLAGS) \
 		--build-arg UNAME_ARCH="$(UNAME_ARCH)" \
 		--build-arg BUILDER_IMAGE="$(BUILDER_IMAGE)" \
 		--tag "$(IMAGE_NAME)" \
-		--network none \
```

Comment:
Note: in order to fetch cargo sources from crates.io upstream _in the image_, we need to allow network access. But since this using the lockfile, this should be ok since typically, on the local machine, the sources would be fetched and mounted into the container.